docs/elements-introduction.adoc

```diff
@@ -10,16 +10,14 @@ layout: page
 
 Vaadin Core Elements is a free and open source set of high quality link:http://webcomponents.org[Web Components] for building mobile and desktop web applications in modern browsers. It builds on top of Google’s link:http://www.polymer-project.org[Polymer] library, and augments the link:https://elements.polymer-project.org[Iron, Paper, and other element sets of Polymer] with elements that are needed in building business applications.
 
-Although based on Polymer, Vaadin Elements can be used together with any other web framework that has support for Web Components. See the  link:integrations/polymer.html[Integrations] section for examples how to use the elements together with different JavaScript frameworks and libraries, such as Angular 2, React and Vue.js.
-
-If you wish to use Vaadin elements with link:http://gwtproject.org[GWT], you can use link:https://vaadin.com/gwt[GWT Polymer Elements].
+Although based on Polymer, Vaadin Elements can be used together with any other web framework that has support for Web Components. See the  link:integrations/polymer.html[Integrations] section for examples how to use the elements together with different JavaScript frameworks and libraries, such as React and Vue.js.
 
 Vaadin Core Elements includes the following elements:
 
  - link:/docs/-/part/elements/vaadin-combo-box/vaadin-combo-box-overview.html[Combo Box]
  - link:/docs/-/part/elements/vaadin-context-menu/vaadin-context-menu-overview.html[Context Menu]
  - link:/docs/-/part/elements/vaadin-date-picker/vaadin-date-picker-overview.html[Date Picker]
- - link:/docs/-/part/elements/vaadin-grid/vaadin-grid-overview.html[(Data) Grid]
+ - link:https://github.com/vaadin/vaadin-grid[(Data) Grid]
```

Comment:
I think we should update all of these links to point to the individual element pages instead (e.g. https://vaadin.com/elements/-/element/vaadin-grid)